packages/react/src/components/Forms/Select/select-menu.tsx

```diff
@@ -28,6 +28,7 @@ export const SelectMenu = ({
     placement: "bottom",
     isOpen,
     maxHeight: 160,
+    shouldFlip: false,
```

Comment:
I wonder if we should not set this setting to true ^^ as when there is no space below the select, the dropdown height is really low and we could not read its content.
Furthermore, we could set `shouldUpdatePosition` to true to resize and flip dropdown on window resize ?